# AUTHOR: Neset Unver Akmandor (NUA)
# E-MAIL: akmandor.n@northeastern.edu

world_frame_name: "world"

pc2_msg_name_sensor1: "/d435_cam1d435/camera/depth/color/points"
pc2_msg_name_sensor2: "/d435_cam2d435/camera/depth/color/points"
pc2_msg_name_sensor3: "/d435_cam3d435/camera/depth/color/points"
pc2_msg_name_sensor4: "/d435_cam4d435/camera/depth/color/points"

bbx_x_min: -1
bbx_x_max: 1
bbx_y_min: -1
bbx_y_max: 1
bbx_z_min: -0.05
bbx_z_max: 0.5

oct_resolution: 0.02
